hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountInfoHandler.java

```diff
@@ -58,13 +106,289 @@ public Response createEmptyResponse(@NonNull final ResponseHeader header) {
     @Override
     public void validate(@NonNull final QueryContext context) throws PreCheckException {
         requireNonNull(context);
-        throw new UnsupportedOperationException("Not implemented");
+        final var query = context.query();
+        final var accountStore = context.createStore(ReadableAccountStore.class);
+        final CryptoGetInfoQuery op = query.cryptoGetInfoOrThrow();
+        if (op.hasAccountID()) {
+            final var account = accountStore.getAccountById(requireNonNull(op.accountID()));
+            validateFalsePreCheck(account == null, INVALID_ACCOUNT_ID);
+            validateFalsePreCheck(account.deleted(), ACCOUNT_DELETED);
+        } else {
+            throw new PreCheckException(INVALID_ACCOUNT_ID);
+        }
     }
 
     @Override
     public Response findResponse(@NonNull final QueryContext context, @NonNull final ResponseHeader header) {
         requireNonNull(context);
         requireNonNull(header);
-        throw new UnsupportedOperationException("Not implemented");
+        final var query = context.query();
+        final var tokensConfig = context.configuration().getConfigData(TokensConfig.class);
+        final var ledgerConfig = context.configuration().getConfigData(LedgerConfig.class);
+        final var accountStore = context.createStore(ReadableAccountStore.class);
+        final var tokenRelationStore = context.createStore(ReadableTokenRelationStore.class);
+        final var tokenStore = context.createStore(ReadableTokenStore.class);
+        final var stakingInfoStore = context.createStore(ReadableStakingInfoStore.class);
+        final var op = query.cryptoGetInfoOrThrow();
+        final var response = CryptoGetInfoResponse.newBuilder();
```

Comment:
Line 126 -132 can be inside the if condition on line 138

Reply:
Will make the change in TokenGetInfoHandler pr.